packages/flame/lib/src/game/game_render_box.dart

```diff
@@ -23,6 +23,9 @@ class GameRenderBox extends RenderBox with WidgetsBindingObserver {
   @override
   bool get sizedByParent => true;
 
+  @override
+  bool get isRepaintBoundary => true;
```

Comment:
This makes the "markNeedsPaint" not to spread to the parent widget, making out 60 fps repaint only affect things inside flame, and not the whole application.